tests/sentry/utils/test_geo.py

```diff
@@ -0,0 +1,24 @@
+from django.test.utils import override_settings
+
+from sentry.testutils.cases import TestCase
+from sentry.testutils.factories import get_fixture_path
+
+
+@override_settings(GEOIP_PATH_MMDB=get_fixture_path("test.mmdb"))
+class TestGeo(TestCase):
```

Comment:
Just want to confirm that this test failed before your changes?

Reply:
Yep.